libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/options/PlayerAttributes.kt

```diff
@@ -0,0 +1,86 @@
+package com.mapbox.navigation.ui.voice.options
+
+import android.media.AudioAttributes
+import android.media.AudioAttributes.CONTENT_TYPE_MUSIC
+import android.media.AudioAttributes.CONTENT_TYPE_SONIFICATION
+import android.media.AudioAttributes.CONTENT_TYPE_SPEECH
+import android.media.AudioAttributes.USAGE_ALARM
+import android.media.AudioAttributes.USAGE_ASSISTANCE_SONIFICATION
+import android.media.AudioAttributes.USAGE_NOTIFICATION_RINGTONE
+import android.media.AudioAttributes.USAGE_VOICE_COMMUNICATION
+import android.media.AudioAttributes.USAGE_VOICE_COMMUNICATION_SIGNALLING
+import android.media.AudioManager
+import android.os.Build
+import androidx.annotation.RequiresApi
+
+/**
+ * PlayerAttributes
+ * An API implements attributes that define how the audio system handles routing
+ * and focus decisions for the specified source.
+ */
+sealed class PlayerAttributes {
```

Comment:
My main worry here is if we're going to need to extend this class (or any of the hierarchy ones) with more attributes in the future, as that would imply breaking SemVer. If we're supporting all the `AudioAttributes` available as of today, we should good though. In other words, not sure if there are any other attributes available apart from `usage`, `contentType` / `streamType`. Refs. https://github.com/mapbox/mapbox-navigation-android/issues/2709